vuln/npm/419.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": 419,
+  "title": "Path Traversal",
+  "overview": "`superstatic` is vulnerable to path traversal on Windows",
+  "created_at": "2018-02-26",
+  "updated_at": "2018-04-29",
+  "publish_date": "2018-04-29",
+  "author": "Сковорода Никита Андреевич (https://github.com/ChALkeR)",
+  "module_name": "superstatic",
+  "cves": [],
+  "vulnerable_versions": "<=5.0.1",
+  "patched_versions": "",
+  "recommendation": "No fix is currently available for this vulnerability.\n\nIt is our recommendation to not install or use this module at this time.",
```

Comment:
Hm…
1. It's fixed in `master` branch.
2. Affects only Windows and certain Node.js versions.

Perhaps the recommendation could be lightened down a bit, e.g.
> The fix is available in the `master` branch, but a new version with the fix has not been released.
> 
> It is our recommendation to not use this module on Windows (and Linux with certain Node.js versions, e.g. v9.9.0) at this time, or to apply the patch manually.

Reply:
Sure, I'll update.

As a policy though do we always want to follow this notion of providing information on non-official releases? It might be easier here because you were involved in this but in other cases if there's no official release as part of the vulnerability report it will not be scalable to track down and check if fixes were made in a repo.

Reply:
@lirantal I think that the difference here is not that I was involved, but that the fix is landed on `master` and there is a release PR at https://github.com/firebase/superstatic/pull/255 (stalled, unfortunately).

Reply:
I agree with updating here but I'm saying it will not be easy for us to follow every project we triage to continually check their repos if they applied a related fix or not.